### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by sbl-plugins**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -545,13 +431,128 @@ MOVE THEM to the space below this block comment and put them in numerical order
...
+5 properties:
+    1) AAX (std::string)
+    2) VST (std::string)
+    3) VST3 (std::string)
+    4) AU (std::string)
+    5) Standalone (std::string)
+3 things it can do:
+    1) function in Pro Tools
+    2) function in Logic
+    3) function in Reaper
```
Comment (matkatmusic) [inline main.cpp]: This is not a good choice for an object.  It is really just a single string that can have 5 values. 
Better choice: DSP engine. 
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
revise number 9, and change #10 accordingly

File: main.cpp (hunk 1 above)
Comment (sbl-plugins) [reply main.cpp]: Completed.

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Cool.  proceed with the rest of the assignment on the same branch

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**